src/mempool/README.md

```diff
@@ -0,0 +1,70 @@
+# Mempool
+
+In Cardano, for blocks to have useful data, they have to contain transactions,
+which are codifications of operations on the Ledger state that only some
+authorized actors can enact. Notice that such transactions are what makes up the
+contents of the block. In one way, one could see the ledger as being able to
+validate transactions, and by implication full blocks as those are mainly
+collections of transactions.
+
+The Mempool is the abstract component of the node that stores transactions which
+are valid on top of the latest known Ledger State (the one at the tip of the
+chain selected by [Chain Selection](../consensus/chainsel.md) in the Consensus
+layer). The validity of transactions is defined by the Ledger layer. Notice a
+change in the selected chain should trigger a prompt revalidation of the pending
+transactions to discard those that became invalid on the new selection.  Such
+collection of valid transactions will be requested by the Consensus layer to
+[forge new blocks](../consensus/forging.md).
+
+Being valid implies in particular that the transaction is pending, i.e. it has
```

Comment:
A valid transaction does not necessarily imply it's pending. Maybe "pending transactions are valid?"

Reply:
Reworded it a bit. Good catch